src/Microsoft.AspNet.Diagnostics.Elm/ElmServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Jetbrains.Annotations;
+using Microsoft.AspNet.Diagnostics.Elm;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class ElmServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Registers an <see cref="ElmStore"/> and configures <see cref="ElmOptions"/>.
+        /// </summary>
+        public static IServiceCollection AddElm([NotNull] this IServiceCollection services, Action<ElmOptions> configureOptions = null)
```

Comment:
Yep.  Default values are ok only for CancellationTokens. Remove ```= null``` from the end and pass it explicitly where needed.